src/v/kafka/protocol/schemata/generator.py

```diff
@@ -32,6 +32,12 @@
 #   ignorable flag on a field doesn't change the wire protocol, but gives
 #   instruction on how things should behave when there is missing data.
 #
+#   - Build a more robust way to define messages and their contents as
+#   sensitive. The current approach relies on a list of root structs to
+#   generate stream operators that don't contain sensitive information. We
+#   should strongly consider leveraging the C++ type system to generate code
```

Comment:
+1 +1

Reply:
Opened https://github.com/redpanda-data/redpanda/issues/4984 to track this